### ISSUE TITLE
Add the possibility to define a maximum number of nodes/tasks that can be executed in parallel for a given Job

### ISSUE THREAD

**Issue by activeeon-bot** [ow2-proactive/scheduling#1054]
<a href="https://jira.activeeon.com/browse/SCHEDULING-1161" title="SCHEDULING-1161">Original issue</a> created by <a href="mailto:guilherme.peretti-pezzi_AT_sophia.inria.fr">Guilherme Peretti Pezzi</a> on 15, Mar 2011 at 16:40 PM - SCHEDULING-1161

<hr />

It would be useful to have the possibility to limit the parallelism of a Job by defining a maximum number of nodes/tasks that can be executed in parallel.

When deploying jobs with lots of independent tasks (typical bag of tasks), such mechanism could be used in order to avoid using all the available resources (for running a low priority job) and would enable to use the scheduler in order to load balance these tasks within a subset of nodes (for benchmarking).
